### PR TITLE
Clarkt/bootstrap v4 release upgrade

### PR DESCRIPTION
**Pull request by clarkt**
Oppdatering til releaseversjon av bootstrap 4

### CODE REVIEW

File: source/_patterns/00-atomer/01-forms/00-avkrysningsboks.mustache [hunk 1]
```diff
@@ -2,25 +2,17 @@
...
-        {{# inactive }}
-          </div>
-        {{/ inactive }}
-      {{/ checkboxes }}
-    </div>
+
+    {{# checkboxes }}
+
+      <div class="custom-control {{#isInline}}custom-control-inline{{/isInline}} custom-checkbox a-custom-checkbox {{# inactive }} inactive {{/ inactive }} {{# isDiscreet }} a-form-checkboxes--discret {{/ isDiscreet }}">
+          <input type="checkbox" {{# isChecked }}checked="checked"{{/ isChecked }} class="custom-control-input" {{# data-toggle }} data-toggle="{{ data-toggle }}"{{/ data-toggle }}{{# data-target }} data-target="{{ data-target }}"{{/ data-target }} id="{{checkboxID}}" required {{# inactive }} disabled {{/ inactive }}>
```
Comment (dhanasneham) [inline source/_patterns/00-atomer/01-forms/00-avkrysningsboks.mustache]: Don't we need option to customize it  like {{# isRequired }} required {{/ isRequired }} ?  This is done in radio button

File: source/_patterns/01-molekyler/05-forms/02-fant-du-det-du-lette-etter.json [hunk 2]
```diff
@@ -6,11 +6,13 @@
 		"radio-class": "a-js-feedbackToggle",
 		"radiobuttons":  [
 			{
+        "isInline" : true,
```
Comment (dhanasneham) [inline source/_patterns/01-molekyler/05-forms/02-fant-du-det-du-lette-etter.json]: Needs some indentation

File: source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json [hunk 3]
```diff
@@ -172,7 +172,7 @@
 	            ]
 	          },
 	          {
-	            "additional-classes": "col-3 hidden-sm-down a-list-sortHeader",
+	            "additional-classes": "col-3 d-none d-sm-block a-list-sortHeader",
```
Comment (dhanasneham) [inline source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json]: Is this not 'd-none d-md-block'?

File: source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json [hunk 4]
```diff
@@ -187,7 +187,7 @@
...
```
Comment (dhanasneham) [inline source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json]: 'd-none d-md-block' right?

File: source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json [hunk 5]
```diff
@@ -236,7 +236,7 @@
 	              ]
 	            },
 	            {
-	              "additional-classes": "col-3 hidden-sm-down",
+	              "additional-classes": "col-3 d-none d-sm-block",
```
Comment (dhanasneham) [inline source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json]: same here

File: source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json [hunk 6]
```diff
@@ -248,7 +248,7 @@
...
```
Comment (dhanasneham) [inline source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json]: same here

File: source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json [hunk 7]
```diff
@@ -267,7 +267,7 @@
 	                  "column-link": {
 	                    "link-class": "a-btn-blue-noUnderline text-right a-fontBold a-fullWidthLink collapsed a-visibleWhenInitial a-hiddenWhenExpanded an-collapse-button pr-0 pr-md-5",
 	                    "link-url": "#",
-	                    "link-text": "<span class='hidden-sm-down'>Utvid</span>",
+	                    "link-text": "<span class='d-none d-sm-block'>Utvid</span>",
```
Comment (dhanasneham) [inline source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json]: d-md-block

File: source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json [hunk 8]
```diff
@@ -277,7 +277,7 @@
 	                  "column-link": {
 	                    "link-class": "a-btn-blue-noUnderline text-right a-fontBold a-fullWidthLink a-visibleWhenExpanded an-collapse-button pr-0 pr-md-5",
 	                    "link-url": "#",
-	                    "link-text": "<span class='hidden-sm-down'>Lukk</span>",
+	                    "link-text": "<span class='d-none d-sm-block'>Lukk</span>",
```
Comment (dhanasneham) [inline source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json]: d-md-block

File: source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json [hunk 9]
```diff
@@ -353,7 +353,7 @@
 	              ]
 	            },
 	            {
-	              "additional-classes": "col-3 hidden-sm-down",
+	              "additional-classes": "col-3 d-none d-sm-block",
```
Comment (dhanasneham) [inline source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json]: d-md-block

File: source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json [hunk 10]
```diff
@@ -365,7 +365,7 @@
...
```
Comment (dhanasneham) [inline source/_patterns/04-sider-altinnett/20-profil/10-profil~personlig.json]: d-md-block

File: source/css/scss-common/_bootstrap.scss [hunk 11]
```diff
@@ -1,32 +1,28 @@
...
-@import './../../node_modules/bootstrap/scss/normalize';
-@import './../../node_modules/bootstrap/scss/print';
-
-// Core CSS
+@import './../../node_modules/bootstrap/scss/root';
 @import './../../node_modules/bootstrap/scss/reboot';
 @import './../../node_modules/bootstrap/scss/type';
 @import './../../node_modules/bootstrap/scss/images';
-/*@import './../../node_modules/bootstrap/scss/code';*/
+// @import './../../node_modules/bootstrap/scss/code';
```
Comment (dhanasneham) [inline source/css/scss-common/_bootstrap.scss]: seems commented. Should we remove this?

File: source/css/scss-common/_bootstrap.scss [hunk 12]
```diff
@@ -39,18 +35,14 @@
 @import './../../node_modules/bootstrap/scss/pagination';
 @import './../../node_modules/bootstrap/scss/badge';
 @import './../../node_modules/bootstrap/scss/jumbotron';
-/*@import './../../node_modules/bootstrap/scss/alert';*/
+// @import './../../node_modules/bootstrap/scss/alert';
```
Comment (dhanasneham) [inline source/css/scss-common/_bootstrap.scss]: to be removed

File: source/css/scss-common/_bootstrap.scss [hunk 13]
```diff
@@ -39,18 +35,14 @@
...
 @import './../../node_modules/bootstrap/scss/close';
-
-// Components w/ JavaScript
 @import './../../node_modules/bootstrap/scss/modal';
 @import './../../node_modules/bootstrap/scss/tooltip';
 @import './../../node_modules/bootstrap/scss/popover';
-/*@import './../../node_modules/bootstrap/scss/carousel';*/
-
-// Utility classes
+// @import './../../node_modules/bootstrap/scss/carousel';
```
Comment (dhanasneham) [inline source/css/scss-common/_bootstrap.scss]: to be removed

File: source/css/style.dist.infoportal-frontpage.scss [hunk 14]
```diff
@@ -1,44 +1,76 @@
...
+@import './../../node_modules/bootstrap/scss/utilities';
+
+// Core variables and mixins
+// @import './../../node_modules/bootstrap/scss/variables';
+// @import './../../node_modules/bootstrap/scss/mixins';
+
+// Reset and dependencies
+// @import './../../node_modules/bootstrap/scss/normalize';
+// @import './../../node_modules/bootstrap/scss/print';
+
```
Comment (dhanasneham) [inline source/css/style.dist.infoportal-frontpage.scss]: Remove the commented lines

File: source/css/scss-common/base/_forms.scss [hunk 15]
```diff
@@ -526,197 +539,134 @@ textarea {
...
-  }
-
-  .a-legend {
-    @include a-fontSize24;
-    color: $black;
-  }
+  display: block;
 
-  .custom-radio {
+  .a-radioButtons-excerpt {
```
Comment (dhanasneham) [inline source/css/scss-common/base/_forms.scss]: Should we also change the name 'custom-radio' in line 674 to 'a-radiobuttons-excerpt'?

File: source/css/scss-common/base/_forms.scss [hunk 16]
```diff
@@ -526,197 +539,134 @@ textarea {
...
-  .custom-control {
-    display: block;
-    padding-bottom: $spacer / 3;
-    padding-left: $spacer * 3;
-    margin-bottom: $spacer;
-    @include media-breakpoint-up( lg ) {
-      display: inline-block;
-      margin-right: $spacer * 2;
-    }
+  .custom-control-input {
```
Comment (dhanasneham) [inline source/css/scss-common/base/_forms.scss]: just wondering if we should change 'custom-control' to 'custom-control-input' in line 696 as well?

File: source/_patterns/00-atomer/01-forms/00-avkrysningsboks.mustache (hunk 1 above)
Comment (clarkt) [reply source/_patterns/00-atomer/01-forms/00-avkrysningsboks.mustache]: Yes, I have fixed this.

**Pull request merged**